src/app/screens/Home/components/Appbar/index.tsx

```diff
@@ -0,0 +1,28 @@
+import i18next from 'i18next';
+import React from 'react';
+import { useHistory } from 'react-router';
+
+import { PATHS } from '~constants/paths';
+import LocalStorageService from '~services/LocalStorageService';
+import { TOKEN_KEY } from '~utils/constants';
+
+import WoloxImg from '../../../Assets/wolox-logo.png';
+
+import styles from './styles.module.scss';
+
+export default function Appbar() {
```

Comment:
ive forgot to mark this but: remember that the export default goes as last line for readability.